nni/retiarii/oneshot/pytorch/proxyless.py

```diff
@@ -136,7 +139,10 @@ class ProxylessTrainer(BaseOneShotTrainer):
     def __init__(self, model, loss, metrics, optimizer,
                  num_epochs, dataset, warmup_epochs=0,
                  batch_size=64, workers=4, device=None, log_frequency=None,
-                 arc_learning_rate=1.0E-3):
+                 arc_learning_rate=1.0E-3, 
+                 grad_reg_loss_type=None, grad_reg_loss_params=None, 
+                 latency_predictor=None, dummy_input=(1, 3, 224, 224),
+                 ref_latency=65):
```

Comment:
What is ref latency?

Reply:
Refer to https://github.com/microsoft/nni/pull/4206#discussion_r716327695